### ISSUE TITLE
Wen Lambo?

### ISSUE THREAD

**Issue by daddys-special-boy** [Marak/colors.js#298]
help, I created an MIT-licensed project and I'm not famous or a billionaire yet 

i am daddy's special boy and deserve better

plz buy my NFT so I can get out of tech, where I clearly don't belong (and do not screenshot it or i will call the cops)

https://opensea.io/assets/matic/0x2953399124f0cbb46d2cbacd8a89cf0599974963/54004601622088790333904220454789765370258842232316711657089034408418645632000

![download](https://user-images.githubusercontent.com/97453754/148778074-2ef48c89-3926-4754-b27f-abe3ea318054.jpeg)

**Comment by karikera** [comment]
it's not a duplicate of #289

**Comment by RIAEvangelist** [comment]
This is a simple problem with a simple solution, either build it yourself or use another lib. Leave the guy alone, this is within his rights according to the license.

Here is an alternative :

`nozaki-colors` https://github.com/RIAEvangelist/nozaki-colors

**Comment by TravkinAlex** [comment]
alt-account cringepost

**Comment by daddys-special-boy** [comment]
> This is a simple problem with a simple solution, either build it yourself or use another lib. Leave the guy alone, this is within his rights according to the license.
> 
> Here is an alternative :
> 
> `nozaki-colors` https://github.com/RIAEvangelist/nozaki-colors

Fortunately, nobody I work with requires petabytes of node_modules to enable colorful emoji debug messages or whatever this library is for. 

Leave him alone? He clearly wants to start a big political discussion. He could have chosen any license he wanted. It's not like there aren't a hundred of them. 

It's like a more childish version of patent trolling, except the patent trolls have more than an MIT license to whine about.

Patent trolls "technically being within their rights" doesn't make them beneficial (or even useful) to the ecosystem.

**Comment by marakdaddysspecial** [comment]
Hi I'm Marak Squires and I your need atention, I'm a legend for my mommy,  just I know how to write stupid things in JS but please follow me, I need to be a billonaire printing emoticons in your console.

Thanks.

![Maraaak](https://user-images.githubusercontent.com/97804104/149633584-87491155-55ac-4fdb-ab51-a3be0674f1c0.png)

**Comment by RIAEvangelist** [comment]
@hollywood this stuff is getting belligerant. Can we do something about this? We are now getting into cyberbullying territory which would be a legal concern for github if Marek was to perform any self-harm and these posts remained.

**Comment by hello-smile6** [comment]
> @Hollywood this stuff is getting belligerant. Can we do something about this? We are now getting into cyberbullying territory which would be a legal concern for github if Marek was to perform any self-harm and these posts remained.

You can always use the report feature on people's accounts. However, if you don't want to, you can always just block people.

**Comment by RIAEvangelist** [comment]
I think this is a moral, social and legal concern for GitHub as a company. A direct liability for them if Marek kills or hurts himself or someone else due to bullying like this. There are laws in place for forum moderation and cyberbullying that could put a legal onus on Github if that happened based on moderation settings.

**Comment by hello-smile6** [comment]
> I think this is a moral, social and legal concern for GitHub as a company. A direct liability for them if Marek kills or hurts himself or someone else due to bullying like this. There are laws in place for forum moderation and cyberbullying that could put a legal onus on Github if that happened based on moderation settings.

I agree, especially considering that there is no way to flag individual comments for moderation without flagging a user at the account level. It seems like everything is expected to be self-moderated, which normally is logical. However, when the maintainer caused the conflict, that obviously will not happen.

**Comment by RIAEvangelist** [comment]
Also, allowing this sort of stuff to stay on the internet is a digital statue to bullying. If no one says anything about it, it will be left to show others that this sort of behavior is ok, and it is not acceptable under any circumstances.

**Comment by hello-smile7** [comment]
@RIAEvangelist 

Enough of defending a nefarious character like Marak Squires. Marak is a psychopath with a colossal ego. He was arrested years ago and has historically always been kicked out of communities (not just github) for unusual behavior. If he does something against him (something a psychopath wouldn't do), it's not because of the community or because of bullying. So please, stop acting like a father protecting an asshole like Marak (he is not a good person); he is a psychopath and a danger to this society and community.

![Marak arrested](https://user-images.githubusercontent.com/97931315/149869236-d16dafd3-ccfd-41c0-b505-cf7c0fb0c2a1.png)

**Criminal history, Marak Squires:**

- **October 2018** Arrested after a fight with his 28-year-old girlfriend, who suffered physical injuries.

- **September 17, 2020**. Arrested with explosive devices and resisting arrest. A neighbor of Queens man with the bomb-making kit: "Obviously, the man is sick."

- **2021** Another act of psychopathy sabotages his JS libraries to affect everyone who depended on her, with zero empathy. He is sick.

* https://abc7ny.com/suspicious-package-queens-astoria-fire/6425363/

* https://www.nydailynews.com/new-york/nyc-crime/ny-queens-bomb-materials-man-charged-20200916-bybbhwqwijbuxg7ndsuy43w6zu-story.html

**Comment by RIAEvangelist** [comment]
I am not defending him. I am defending our civil culture.

Despite him being what he is, it is not an excuse for people in our community to behave like animals. Inorder to have a civil community, it means we must be civil even when others are not.

The is nothing wrong with speaking the truth, but calling people names and then attacking other members of the community like some have done in several threads does nothing but spread discord, resentment and undermine the fabric of our shared desire to help with OSS and free software.

I know it is hard to see why we must be civil when others are not, but this is the nature of being the better person when times are tough. If we don't each do our part in that then it will be a black eye not just for someone like Marek, but also for our community as a while for how we handle our problems.

There is a code of conduct for just this reason and part of that is that you need to be polite and over 13, aka mature. Maturity is required for many reasons, including the ability to asses what sort of behavior is acceptable.

Again I do not defend what Marek did, just the standards I hold our community to when things like this happen. Several people have been banned for badgering more than just Marek and for harassing and encouraging harassing behavior. That is the real issue here.

What Marak did has exposed the best and worst of our community. We would be remiss to not address the bad things that have come from this to help prevent it in the future.

**Comment by bobmartins79** [comment]
![0c3afc2522b87911c590d7dc0b564c4a](https://user-images.githubusercontent.com/97931315/150055862-3d53e4b4-7371-4928-b1a7-227093965957.jpg)

**Comment by RIAEvangelist** [comment]
Wow that's an extremely divisive comment, seemingly meant to antagonize. The level of intelligence required to write it in such a way is also impressive. Thank you for putting that level of consideration into your commenting. It shows wonderful passion.

Being civil to one another is not something we should look the other way about. Name calling, and bullying are best left at the schoolyard and have no place in a community like GitHub. That's what Facebook and reddit are for.

Github's actions show their intent to encourage others to follow the code of conduct laid out with the terms of service of the platform.

When they remove the parts that encourage us to be a civil community inspiring growth and compassion, for one another, not just the repo owner, then it's free reign to act like recess time. Until then the code of conduct and terms of service here dictate that we carry ourselves as somewhat mature humans attempting to be decent.

**Comment by bobmartins79** [comment]
:heartbeat: :heartbeat: :kissing::kissing::v:

**Comment by RIAEvangelist** [comment]
I'm quite happy here thanks.

**Comment by bobmartins79** [comment]
Nice. 

So, happy coding.

Bye, bye.

**Comment by bobmartins79** [comment]
> Happy coding @bobmartins79, and for anyone reading this, [my response to the social aspects of the previous commentary on all of us being civil is in this linked thread](https://github.com/Marak/colors.js/issues/302#issuecomment-1016612515). It relates to how we decide to interact with people in real life and online. We are all people, even if we are just behind a computer screen.

LOL.

**Comment by RIAEvangelist** [comment]
Anyone reading this, it seems like I am talking to myself in the chain, however, the individuals, and their posts that were creating an abusive and toxic community have been dealt with. Apologies for any inconvenience and props to @github for helping promote a healthy and encouraging community where we can all code in the true spirit of free and open-source collaboration!
